src/dealDashboard/dealInfo/dealInfo.html

```diff
@@ -144,9 +144,36 @@ <h1 class="title heading gradient">Deal Information</h1>
       </etherscanlink>
     </div>
 
+    <div if.bind="deal.registrationData.terms.daoplomatRewards"
+      class="infoSection totalRepresentatives ${showMoreRewards ? 'active' : ''}"
+      click.trigger="showMoreRewards = !showMoreRewards"
+    >
+      <span class="bodySmallTextBold infoTitle">Total DAOplomat Reward</span>
+      <span>
+        <span class="reward-percent">
+          ${deal.registrationData.terms.daoplomatRewards.percentage}
+        </span>
+        <i class="fas fa-angle-down arrow"></i>
+      </span>
+    </div>
+    <div class="panel" css.bind="{'max-height': maxHeightRewards}" ref="panelRewards">
+      <div class="panel-content">
+        <p
+          class="representative bodySmallText"
+          repeat.for="daoplomat of deal.registrationData.terms.daoplomatRewards.daoplomats"
+        >
+          <etherscanlink 
+            class="representative-dao" 
+            address.bind="daoplomat.address"
+            text="${daoplomat.address | smallHexString}"></etherscanlink>
+          <span class="reward-percent">${daoplomat.splitPercentage * deal.registrationData.terms.daoplomatRewards.percentage / 100}</span>
```

Comment:
```suggestion
          <span class="reward-percent">${daoplomat.rewardPercentage * deal.registrationData.terms.daoplomatRewards.percentage / 100}</span>
```

I understand that the idea behind "split", but I'd prefer to stick to "reward", the business term.

Reply:
How about a compromise, `rewardSplitPercentage`, to emphasize the relationship to the total reward percentage?

Reply:
Renamed to `rewardSplitPercentage`, as discussed with @hiaux0.

@IonelLupu FYI